add new files for verification

SergeyRZN opened the pull request:
Please check and write the result of verification.

SergeyRZN committed:
add new files for verification

aneelia commented:
текст коммита не соответствует действительности, я об этом рассказывала. это важно

aneelia commented on src/ex2_js-basics-part2/task-01.js:
```diff
@@ -0,0 +1,12 @@
+"use strict";
+function checkType(valueParametr) {
+    switch (typeof(valueParametr)) {
```
здесь нет смысла использовать свич, он не оптимизирует код, только лишние строки
typeof(valueParametr) - можно без скобок

aneelia commented on src/ex2_js-basics-part2/task-03.js:
```diff
@@ -0,0 +1,26 @@
+    let notEventNumber = 0, eventNumber = 0, zeroNumber = 0, arrayNumber;
+    for (let i = 0; i < myArray.length; i++) {
+        if(typeof(myArray[i]) !== "number") {
+          continue;
```
1) continue не обязательно, вообще на практике ни разу не видела
2) бесполезная проверка

SergeyRZN replied:
Просила написать на счет проверка типа элемента массива.
Если элемент массива не число, то будет брать следующий элемент массива. Это проверка была в задании, вот кусок текста из задания № 3 - 
"При выполнении задания необходимо учесть, что массив может содержать не только числа, но и, например, знаки, null и тд."

SergeyRZN replied:
Просила написать на счет проверки типа элемента массива. Если элемент массива **НЕ** число, тогда берем следующий элемент массива.  Это проверка была описана в задании №3. Вот кусок текста из задания №3 - 
«При выполнении задания необходимо учесть, что массив может содержать не только числа, но и, например, знаки, null и тд.».

aneelia replied:
да, спасибо

aneelia commented on src/ex2_js-basics-part2/task-03.js:
```diff
@@ -0,0 +1,26 @@
+          zeroNumber = zeroNumber + 1;
+          continue;
+        };
+        switch (myArray[i] % 2) {
```
здесь свич не нужен, подумай как переделать алгоритм на более простой

aneelia commented on src/ex2_js-basics-part2/task-03.js:
```diff
@@ -0,0 +1,26 @@
+        notEventNumber = notEventNumber +1;
+        }
+    }
+    console.log (eventNumber);
```
консоли ни к чему

aneelia commented on src/ex2_js-basics-part2/task-06.js:
```diff
@@ -0,0 +1,26 @@
+    }
+	outerWhile:
+    for (let i = 2; i < valueParametr; i++) {
+      switch (valueParametr % i) {
```
не нужен свич, тут всего 2 условия - 0 и дефолтное
в таких случаях нужен if

aneelia commented on src/ex2_js-basics-part2/task-06.js:
```diff
@@ -0,0 +1,26 @@
+"use strict";
+function chekSimpleNumber(valueParametr) {
+    let resultat;
```
учим инглиш - result

SergeyRZN committed:
Changed the switch construct to if

SergeyRZN committed:
Replaced switch with if. Removed the console call

SergeyRZN committed:
Replaced switch with if. Replaced the variable resultat with result

aneelia commented on src/ex2_js-basics-part2/task-01.js:
```diff
@@ -0,0 +1,8 @@
+"use strict";
+function checkType(valueParametr) {
+    if (typeof valueParametr === "number" || typeof valueParametr === "string") {
```
нужна проверка на NaN

aneelia commented on src/ex2_js-basics-part2/task-02.js:
```diff
@@ -0,0 +1,10 @@
+"use strict";
+function arrayCount(myArray) {
+    let countArray = 0;
```
эта переменная не нужна
есть свойство length для таких целей, которое ты используешь....

aneelia commented on src/ex2_js-basics-part2/task-03.js:
```diff
@@ -0,0 +1,22 @@
+"use strict";
+function checkNumber(myArray) {
+    let notEventNumber = 0, eventNumber = 0, zeroNumber = 0, arrayNumber;
```
мы тут англичане все собрались
так что давай наверно нормально напишем - even, odd, zero

aneelia commented:
хотелось бы увидеть правки

SergeyRZN committed:
Task-01 added a check on NAN. Task-02
deleted a variable (countArray). Task-03
changed the variables (notEventNumber) to (odd), (eventNumber) to (event), (zeroNumber) to (zero)
